Makefile

```diff
@@ -98,7 +98,10 @@ scap-scan:
 	unzip -p scap-security-guide-${open_scap_version}.zip scap-security-guide-${open_scap_version}/ssg-rhel8-ds.xml > scap/ssg-rhel8-ds.xml
 	docker run -itd --name scap-scan -v $(PWD)/scap:/scap ${current_image}
 	docker exec -u root scap-scan /bin/bash -c "microdnf install -y openscap-scanner"
-	docker exec -u root scap-scan /bin/bash -c "oscap xccdf eval --profile xccdf_org.ssgproject.content_profile_cis --results /scap/scap_scan_results.xml --report /scap/scap_scan_report.html /scap/ssg-rhel8-ds.xml > /scap/command-output.txt 2>&1" || true
+	# ensure the file is owned by root in order to avoid permission issues
+	docker exec -u root scap-scan /bin/bash -c "chown root:root /scap/ssg-rhel8-ds.xml"
+	# UBI-minimal images do not have the authselect package installed so we skip xccdf_org.ssgproject.content_rule_enable_authselect rule to avoid false positive
+	docker exec -u root scap-scan /bin/bash -c "oscap xccdf eval --profile xccdf_org.ssgproject.content_profile_cis --results /scap/scap_scan_results.xml --report /scap/scap_scan_report.html --skip-rule xccdf_org.ssgproject.content_rule_enable_authselect /scap/ssg-rhel8-ds.xml > /scap/command-output.txt 2>&1" || true
```

Comment:
As discussed we do not want to exclude rules.
Do not merge the PR right now until we get an answer from RHEL